libqtile/core/manager.py

```diff
@@ -189,8 +190,9 @@ def noop(qtile):
         hook.fire("startup")
 
         if state:
-            st = pickle.load(io.BytesIO(state.encode()))
             try:
+                with open(state, 'rb') as f:
+                    st = pickle.load(f)
```

Comment:
I think we need to remove the file here.

Reply:
I forgot to add this to the update I just pushed... sigh. I omitted it from the initial PR on the basis that "it will be removed when the system shuts down" and "maybe someone will want to save it elsewhere for usage", but I guess not all systems use a tmpfs for /tmp, and I doubt anybody would want it. I'll update that now.